packages/ui-core/src/components/Select/Select.tsx

```diff
@@ -171,63 +190,35 @@ class Select extends React.Component<ISelectProps, ISelectState> {
         }
 
         this.setState((state) => ({ isOpen: !state.isOpen }));
+        this.onEventListener();
     }
 
     handleChangeCombobox = (e: React.ChangeEvent<HTMLInputElement>): void => {
         const { items } = this.props;
+        const filterValue = e.target.value;
 
-        const currentItems = items.filter(item => {
-            if ( e.target.value.length <= item.title.length) {
-                return RegExp(e.target.value, 'ig').test(item.title);
+        const filteredItems = items.filter(({ title }) => {
+            if (filterValue.length <= title.length) {
+                return RegExp(filterValue, 'ig').test(title);
             }
 
-            return;
+            return false;
         });
 
         this.setState({
             activeIndex: 0,
             isOpen: true,
-            comboboxItems: currentItems,
-            inputValue: e.target.value,
+            comboboxItems: filteredItems,
```

Comment:
Имелось ввиду переименование переменной в стейте, чтобы тут писать
```suggestion
            filteredItems,
```